README.md

```diff
@@ -12,18 +12,18 @@ See the [wiki](https://wiki.alphaloop.ch/) for more information on the project.
 ### > Installation using `pip`
 ```
 pip install gammaloop
+gammaloop --build_dependencies
+source `gammaloop -show_venv`
```

Comment:
Very minor nitpick: isn't show_venv a bit too wordy for a short flag?

Reply:
right indeed, there is anyway a long verbose option with double dash. Will shorten it to `-venv`